parent/pom.xml

```diff
@@ -571,5 +571,24 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>parallel-tests</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <forkCount>1C</forkCount>
+              <reuseForks>true</reuseForks>
+              <systemPropertyVariables>
+                <!-- without the leading 0 the property was not expanded -->
+                <testForkNumber>0${surefire.forkNumber}</testForkNumber>
```

Comment:
I'm not sure if I get this comment :see_no_evil:

Reply:
I had the issue that when I only had the surefire property inside the system property, i.e. `<testForkNumber>${surefire.forkNumber}</testForkNumber>`, that then the property `testForkNumber` was empty on test execution. I didn't understood why but also didn't investigated further. So as a workaround I added a `0` as prefix as then the property is set and still parseable as integer.

Reply:
ok weird :sweat_smile: http://maven.40175.n5.nabble.com/SureFire-2-16-doesn-t-respect-forkCount-tp5771850p5771854.html

Reply:
Oh thanks for this I will then change it to `$${` instead of `0${` as this makes more sense :+1: